celery/app/defaults.py

```diff
@@ -180,6 +180,7 @@ def __repr__(self):
             type='float', old={'celery_task_result_expires'},
         ),
         persistent=Option(None, type='bool'),
+        extended=Option(True, type='bool'),
```

Comment:
Nice! :smile: 

Just a note though; as also noted below, I believe this should be set to `False` by default in order to maintain current behavior.

Reply:
Done. Thanks!